.pre-commit-config.yaml

```diff
@@ -0,0 +1,40 @@
+repos:
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v3.2.2
+    hooks:
+      - id: pyupgrade
+        args: [--py37-plus]
+
+  - repo: https://github.com/psf/black
+    rev: 22.10.0
+    hooks:
+      - id: black
+        args: [--target-version=py37]
+
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+
+  - repo: https://github.com/PyCQA/flake8
+    rev: 5.0.4
+    hooks:
+      - id: flake8
+        additional_dependencies: [flake8-2020]
```

Comment:
Flake8 to find style errors using pycodestyle and pyflakes (both builtin plugins: https://flake8.pycqa.org/en/latest/) and flake8-2020 (version comparison problems: https://github.com/asottile/flake8-2020)

Reply:
Maybe worth replacing `flake8-2020` with something much more useful, [flake8-bugbear](https://github.com/PyCQA/flake8-bugbear). It's more or less the official set of first-party optional checks by the PyCQA that don't meet the strict standards of `pyflakes` but are still very likely to be mistakes of some sort; I'm not sure I've ever seen a false positive with it. It seems a lot more worth adding than `flake8-2020`, which is a single check for a single specific issue that is not used here and very unlikely to ever be, at the point.